client/src/views/client-area/statistics/StatisticsPage.vue

```diff
@@ -238,7 +238,129 @@
+          :animated="true"
+          class="skeleton-loading-title"
+        />
+        <div class="users-cards-list">
+          <ion-card
+            class="users-cards-list-item"
+            v-for="index in 3"
+            :key="index"
+          >
+            <div class="users-cards-list-item-icons">
+              <ion-icon
+                class="users-cards-list-item-icons__main"
+                :icon="person"
+              />
+            </div>
+            <div class="users-cards-list-item-text">
+              <ion-skeleton-text
+                :animated="true"
+                class="skeleton-loading-number"
+              />
+              <ion-skeleton-text
+                :animated="true"
+                class="skeleton-loading-text"
+              />
+            </div>
+          </ion-card>
+        </div>
+      </div>
+      <!-- revoked users -->
+      <div class="users revoked skeleton">
+        <ion-skeleton-text
+          :animated="true"
+          class="skeleton-loading-title"
+        />
+        <div class="users-cards-list">
+          <ion-card
+            class="users-cards-list-item"
+            v-for="index in 3"
+            :key="index"
+          >
+            <div class="users-cards-list-item-icons">
+              <ion-icon
+                class="users-cards-list-item-icons__main"
+                :icon="person"
+              />
+            </div>
+            <div class="users-cards-list-item-text">
+              <ion-skeleton-text
+                :animated="true"
+                class="skeleton-loading-number"
+              />
+              <ion-skeleton-text
+                :animated="true"
+                class="skeleton-loading-text"
+              />
+            </div>
+          </ion-card>
+        </div>
+      </div>
+      <!-- storage -->
+      <div class="storage skeleton">
+        <ion-skeleton-text
+          :animated="true"
+          class="skeleton-loading-title"
+        />
+        <div class="storage-data">
+          <div class="storage-data-global">
+            <ion-skeleton-text
+              :animated="true"
+              class="skeleton-loading-number"
+            />
+            <div class="storage-data-global-info">
+              <ion-skeleton-text
+                :animated="true"
+                class="skeleton-loading-text"
+              />
+              <ion-skeleton-text
+                :animated="true"
+                class="skeleton-loading-text"
+              />
+              <ion-skeleton-text
+                :animated="true"
+                class="skeleton-loading-text"
+              />
+            </div>
+          </div>
+          <div class="storage-data-consumption">
+            <ion-skeleton-text
+              :animated="true"
+              class="skeleton-loading-text"
+            />
+            <div class="storage-data-consumption-content">
+              <div
+                v-if="firstBarData"
```

Comment:
`firstBarData` will result to undefined while loading, you can remove this `v-if`. We should always display at least the first bar.